mkdocs.yml

```diff
@@ -1,11 +1,23 @@
 docs_dir: docs/book
 site_dir: docs/html
+extra:
+    project: Components
+    base_url: https://docs.laminas.dev/
+    project_url: https://docs.laminas.dev/components/
```

Comment:
Can be removed because the build script sets these options.

Reply:
Done.